addon/player.js

```diff
@@ -52,11 +52,54 @@ class Player extends BasePlayer {
     this.moduleName = moduleName;
     this.testName = testName;
     this.success = true;
+    this.pausePromise = null;
+    QUnit.config.testTimeout = undefined;
```

Comment:
Let's keep QUnit configurarion on the vendor file only, so the code is independent of the testing franework